### DISCUSSION TITLE
How can I hide/remove the app icon from the dock(macOS) while keeping it in the menu bar?

### DISCUSSION THREAD

**Discussion by Jun10ng** [tauri-apps/tauri#6038] [Q&A]
How can I hide/remove the app icon from the dock(macOS) while keeping it in the menu bar?

**Comment by rainbowatcher** [comment]
you can try this
```rust
tauri::Builder::default().setup(|app| 
app.set_activation_policy(tauri::ActivationPolicy::Accessory));
```

**Reply by mdings** [reply]
I _think_ for v2 it as following:

```rust
tauri::Builder::default().setup(|app| 
  #[cfg(target_os = "macos")]
  app.set_activation_policy(tauri::ActivationPolicy::Accessory);
);
```

**Reply by thiagovianav** [reply]
> I _think_ for v2 it as following:
> 
> ```rust
> tauri::Builder::default().setup(|app| 
>   #[cfg(target_os = "macos")]
>   app.set_activation_policy(tauri::ActivationPolicy::Accessory);
> );
> ```

If I want to use it in only one window what should I do?

I just want to use it in my secondary invisible window.

I'm bad at Rust so sorry for bottering

**Reply by FabianLars** [reply]
I think converting the window to an [NSPanel](https://github.com/ahkohd/tauri-nspanel) would show that behavior? I don't think it's possible to hide a "normal" window from the dock but not sure (couldn't find anything at least)
